sgkit/distance/api.py

```diff
@@ -8,12 +8,14 @@
 from sgkit.typing import ArrayLike
 
 MetricTypes = Literal["euclidean", "correlation"]
+DeviceTypes = Literal["cpu", "gpu"]
 
 
 def pairwise_distance(
```

Comment:
Can you also add this to the top-level API in https://github.com/pystatgen/sgkit/blob/master/sgkit/__init__.py?